clusters/templates/catalog/iceberg-trino.properties

```diff
@@ -0,0 +1,20 @@
+connector.name=iceberg
+
+hive.metastore.uri=thrift://a74b6e20f132e4eac840a591d5caf0ba-c3bc190f0aee9cb5.elb.us-east-1.amazonaws.com:9083
+hive.s3.ssl.enabled=true
+hive.s3.path-style-access=true
+hive.s3.iam-role=arn:aws:iam::093347738777:role/deploy-cluster-instance-s3-access-role
+
+# metastore
+hive.metastore-cache-ttl=240m
+hive.metastore-refresh-interval=30m
+hive.metastore-cache-maximum-size=1000000
+# writer properties
+iceberg.max-partitions-per-writer=5000
+
+# metadata caching
+hive.parquet.use-column-names=true
+hive.parquet-batch-read-optimization-enabled=true
+iceberg.parquet.metadata-cache-enabled=true
```

Comment:
Actually I am not able to find this in Trino's source code..
https://github.com/search?q=repo%3Atrinodb%2Ftrino+metadata-cache-enabled&type=code